content/posts/blog/first-paris-xmpp-meetup.md

```diff
@@ -6,8 +6,8 @@ blog_id: blog
 category: Summits, Conferences, And Meetups
 ---
 
-After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
+After Berlin, Stockholm, and London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
 
-The Paris meetup is very simply organised by the JabberFr community, as this will be mostly oriented discussions our the modernity and contemporaneity of XMPP/Jabber.
+The Paris meetup is simply organised by the JabberFr community, as this will be mostly oriented discussions on the modernity of XMPP/Jabber.
```

Comment:
`Sorry, I think I wasn't clear; "simply organised" still doesn't make sense to me. What does it mean to organize something simply?
`
Without waste, very few preparation, nothing fancy like a slidedeck, just two subjects and the open discussions that go around.

`Also, modernity does not mean "discussing modern XMPP", modernity is a post-medieval historical period.
`

Oh I see, thanks for explaining. Well at least post-medieval is no medieval anymore, which is an improvement. ;-)